src/test/java/com/minecolonies/util/DistanceSquaredTest.java

```diff
@@ -0,0 +1,73 @@
+package com.minecolonies.util;
+
+import com.minecolonies.colony.ColonyManager;
+import net.minecraft.util.BlockPos;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+import java.util.Random;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.number.OrderingComparison.greaterThanOrEqualTo;
+
+@PrepareForTest({ColonyManager.class, LanguageHandler.class})
+@RunWith(PowerMockRunner.class)
+public class DistanceSquaredTest
+{
+
+    /**
+     * Max x/z position in vanilla minecraft.
+     */
+    private static final int MAX_POSITION =  30_000_000;
+
+    @Test
+    public void test2DTwoPositions()
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this method name to match the regular expression: '^test[A-Z][a-zA-Z0-9]*$' [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS3578)